sycl/source/detail/kernel_bundle_impl.hpp

```diff
@@ -350,17 +359,30 @@ class kernel_bundle_impl {
                     std::string *LogPtr) {
     assert(MState == bundle_state::ext_oneapi_source &&
            "bundle_state::ext_oneapi_source required");
-    assert(Language == syclex::source_language::opencl &&
-           "TODO: add other Languages. Must be OpenCL");
-    if (Language != syclex::source_language::opencl)
+    assert((Language == syclex::source_language::opencl ||
```

Comment:
I think I was mistaken when I originally put this assert in here. We throw an exception if the language isn't supported. And `language` is an enumerated type, so it's not really possible for it to be any other value.  I think this whole assertion of language value is redundant and should be removed.